build/args.js

```diff
@@ -0,0 +1,11 @@
+module.exports = require('yargs')
```

Comment:
there is this explicit dependency on `yargs` but it's not in the package.json listed under devDependencies. Guess we should add this right? Which version is needed?

Reply:
yes, missed this. Probably latest